### DISCUSSION TITLE
A repository for Flatimages here on Github

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [flatimage/flatimage#7] [Ideas]
talvez seria interessante fazer um repositório aqui no github tipo assim: https://github.com/pacstall/pacstall-programs/tree/master/packages e os usuários poderiam fazer um pull request para adicionar os programas e etc... assim daria para fazer um programa CLI para puxar os programas desse repositório e instalar

**Comment by ruanformigoni** [comment]
Bacana! Vou fazer isso :smile:

**Comment by Heus-Sueh** [comment]
alguns programas que seriam interessantes fazer um flatimage:

lutris - a versão flatpak é pouco mantida
anydesk - a versão flatpak foi descontinuada
